### PR TITLE
Updates to Testing

### PR DESCRIPTION
**Pull request by Dargones**
This pull request brings back some of the tests that are part of this repository but are not executed with GitHub workflow. I am hoping that integrating the tests with GitHub CI would make it easier to sync Corral with Boogie and to otherwise maintain/bring back the project. Here is the detailed change-log:

- Removed `net5.0` target in favor of `net6.0` since Corral no longer passes the main regression suit when built on `net5.0`.
- Fixed paths in two `.csproj` files so that Angelic Verifier compiles; added compilation of Angelic Verifier as a step in the CI.
- Added steps to the CI to execute `AliasAnalysis` and `fwdbck` tests on Linux.
- Removed `test/AliasAnalysis/f8.bpl` since (as far as I can tell) this test has never been executed, with the [`runtest.bat` jumping from `f7.bpl` to `f9.bpl`](https://github.com/boogie-org/corral/blob/030bf6c3afc17e79751641c1f7a28794dd7f5e0f/test/AliasAnalysis/runtest.bat#L6). There is also no expected output information for this test.
- Removed `test/choudini` directory since my understanding is that Concurrent Houdini has been removed as of [this commit](https://github.com/boogie-org/corral/commit/0ce370105fb1348a9200c2f8dc80a58f5601d298).
- Removed `test/old-regressions/map/map.bpl` and `test/dual/f1.bpl` because there seem to be no golden standard for these tests.

I also have a few notes\questions about potential next steps. @zvonimir, would you have any advice on this?

- I was able to write bash scripts that run `av-regressions` and `propinst-regressions` but the results do not match with what is in the repo. Based on my experiments, the results change with different versions of z3, so I am wondering if the golden standard currently in the repository is for a version of z3 different from 4.8.8, which is [the one that the CI uses](https://github.com/boogie-org/corral/blob/030bf6c3afc17e79751641c1f7a28794dd7f5e0f/.github/workflows/test.yml#L7). Would be grateful for any hints on how to approach this!
- I have not yet tried bringing back the `old-regressions` tests. Would that be worth trying, i.e. are they likely to still work with the current version of Corral and support later Boogie versions?
- I am hoping to make another PR that would make the tests platform-independent and add Mac and Windows to the CI once I figure out the set of all active tests. I am thinking to use lit for this purpose but am not sure.

### CODE REVIEW

**Comment by akashlal** [comment]
The changes look good. Regarding your questions, the only tests that we should look to keep at the moment are `regression` and `av-regressions`. Corral has many research/experimental features that are likely no longer required. I suspect that `av-regressions` need some care as well. For now, we can update the golden output for these tests -- semantically, the current output looks correct.

I'll try to be more responsive to any issues or PRs that you file.

**Comment by akashlal** [comment]
I also tried with 4.8.8 and noticed the same with AV results.

**Review by akashlal — APPROVED** [review]

File: test/av-regressions/run.sh [hunk 1]
```diff
@@ -1,23 +1,13 @@
...
+  echo "Succeeded"
+  exit 0
 else
-    echo Failed
-    exit 1
-fi
-
+  echo "FAILED"
+  exit 1
+fi
```
Comment (rakamaric) [inline test/av-regressions/run.sh:13]: nit: let's add newline in the end of all these files that are showing this warning. I think that is a good practice.

**Review by rakamaric — CHANGES_REQUESTED** [review]

File: test/av-regressions/run.sh (hunk 1 above)
Comment (Dargones) [reply test/av-regressions/run.sh]: Done!

**Pull request merged**